test/topo/topo.go

```diff
@@ -83,3 +115,69 @@ func (s *TestSuite) TestAddRemoveDevice(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Equal(t, len(res.Objects) == 1 && res.Objects[0].ID == "2", true)
 }
+
+func (s *TestSuite) TestScale(t *testing.T) {
```

Comment:
We can move this to a separate file like e2t tests in the current folder.

Reply:
Good idea. Done.